packages/react-celo/src/utils/local-storage.ts

```diff
@@ -79,36 +115,26 @@ export function setTypedStorageKey<
   localStorage.setItem(key, value.toString());
 }
 
-export function removeLastUsedAddress() {
-  localStorage.removeItem(localStorageKeys.lastUsedAddress);
-}
-
 export type WalletArgs =
   | [string]
   | [CeloTokenContract]
   | [WalletConnectWalletOptions]
   | [number]
   | [];
 
-export function getLastUsedWalletArgs(): WalletArgs | null {
-  const args = localStorage.getItem(localStorageKeys.lastUsedWalletArguments);
-  if (args && args.length) {
-    const parsed = JSON.parse(args) as WalletArgs;
-
-    return parsed;
-  }
-
-  return null;
-}
-
 export function clearPreviousConfig(): void {
   Object.values(localStorageKeys).forEach((val) => {
     if (val === localStorageKeys.lastUsedWalletId) return;
     if (val === localStorageKeys.lastUsedWalletType) return;
+    if (val === localStorageKeys.lastUsedWallets) return;
     localStorage.removeItem(val);
   });
 }
 
+export function wipeStorage() {
+  localStorage.clear();
+}
+
```

Comment:
Is this used?

Reply:
only in the tests to make sure its completely reset